src/benchmarks/micro/Serializers/Xml_FromStream.cs

```diff
@@ -18,6 +19,7 @@ namespace MicroBenchmarks.Serializers
     [GenericTypeArguments(typeof(XmlElement))]
     [GenericTypeArguments(typeof(SimpleStructWithProperties))]
     [GenericTypeArguments(typeof(ClassImplementingIXmlSerialiable))]
+    [AotFilter("Currently not supported due to missing metadata.")]
```

Comment:
Issues caused by missing metadata can be fixed with `rd.xml`.

Reply:
@hez2010 great point! I've opened https://github.com/dotnet/performance/issues/2338 to track it